### DISCUSSION TITLE
Public Discussion

### DISCUSSION THREAD

**Discussion by 19Si** [The-Basement-Dwellers/DIT-A91906#39] [General]
### **Public Discussion**
This is a School Assessment, we would appreciate the feedback but please don't give improved code or any code as it will invalidate the assessment.

you can supply us with general feedback like:
- what features are not functional, or janky, or unnecessary 
- what would be cool to add
- what could we change?

Thanks for checking out our project! 😄

**Comment by Nuddley** [comment]
The door should have an animation, It's hard to tell if its open or not, also a way to tell when your being attacked. The health bar is confusing coming from the middle and the red seems to be the health you have left and not the damage taken. Also the camera aim is a bit odd using the keys a mouse aim would've made sense. Also moving the charge bar to a HUD style instead as I initially thought it was a shield.
Very good game and I look forward to having the speed run record when its complete.

**Reply by taiyotaguchi** [reply]
Attacking is tedious. Good start though
